core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/show-message-utils.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Utility function for logging the progress of the tests.
+ */
+
+let showMessage = async function(message) {
+  await console.log(message); // eslint-disable-line no-console
+};
+
+module.exports = { showMessage };
+
+/** We need to ignore the eslint checks for the console statements in our
```

Comment:
I think this message should go above line 20. It suffices to say "We use console logging statements in the Acceptance Tests to log the progress of, or feedback from, the tests."

Reply:
done